<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SpellChecker</class>
 <widget class="QDialog" name="Dialog_SpellChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>701</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>701</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaTeX Office Writer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditReplaceWith"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelUnknownWord">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUnknown">
     <property name="text">
      <string>Unknown word:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelReplace">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAddToDictionary">
       <property name="text">
        <string>Add to dictionary</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReplaceOnce">
       <property name="text">
        <string>Replace once</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReplaceAll">
       <property name="text">
        <string>Replace all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonIgnoreOnce">
       <property name="text">
        <string>Ignore once</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonIgnoreAll">
       <property name="text">
        <string>Ignore all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>240</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>180</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="textEditDocument">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelDocument">
     <property name="text">
      <string>Document:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEditDocument</tabstop>
  <tabstop>lineEditReplaceWith</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButtonAddToDictionary</tabstop>
  <tabstop>pushButtonReplaceOnce</tabstop>
  <tabstop>pushButtonReplaceAll</tabstop>
  <tabstop>pushButtonIgnoreOnce</tabstop>
  <tabstop>pushButtonIgnoreAll</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
